lib/web_ui/dev/test_runner.dart

```diff
@@ -404,13 +460,20 @@ class TestCommand extends Command<bool> with ArgUtils {
           'BUILD_MAX_WORKERS_PER_TASK': io.Platform.environment['BUILD_MAX_WORKERS_PER_TASK'],
       },
     );
-    stopwatch.stop();
-    print('The build took ${stopwatch.elapsedMilliseconds ~/ 1000} seconds.');
 
     if (exitCode != 0) {
       throw ToolException(
           'Failed to compile tests. Compiler exited with exit code $exitCode');
     }
+
+    if (forCanvasKit) {
+      final io.Directory canvasKitTemporaryOutputDirectory = io.Directory(path.join(environment.webUiRootDir.path, canvasKitOutputRelativePath, 'test', 'canvaskit'));
+      final io.Directory canvasKitOutputDirectory = io.Directory(path.join(environment.webUiBuildDir.path, 'test', 'canvaskit'));
+      if (await canvasKitOutputDirectory.exists()) {
```

Comment:
nit: can we utilize the temporary directories. Example: 

https://github.com/flutter/engine/blob/703525502b19dce99674f2425375b5371e3ff59d/lib/web_ui/dev/utils.dart#L208

You can also add a cleanup callback.

Reply:
What would the clean-up callback do? I'm not cleaning up anything temporary here AFAICT.

Reply:
If I'm not mistaken, in this PR, you are creating a directory for  canvasKitOutput. This directory is created as temporary directory (not a cipd package directory for example) so better to be cleaned up before felt finishes (during the cleanup).

The pattern we recently developed in felt is adding these type of temporary directories to ```temporaryDirectories``` (https://github.com/flutter/engine/blob/master/lib/web_ui/dev/utils.dart#L197) and cleaning all of them when felt ends.

Reply:
Ah, no, we want to keep compiled tests because it allows `build_runner` to rebuild tests incrementally (e.g. recompile only one test when running multiple tests). We also frequently visually inspect the contents of the JS files, for example to check that code tree shaking works properly.

Reply:
thanks for the details.